### PR TITLE
Re-enable main Facility Locator Cypress test

### PR DESCRIPTION
**Pull request by brangi**
## Description
issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/20178

## Acceptance criteria
- [x] Re-enable main Facility Locator Cypress test

### CODE REVIEW

File: src/applications/facility-locator/tests/e2e/search.cypress.spec.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ Cypress.Commands.add('verifyOptions', () => {
 describe('Facility search', () => {
   before(function() {
     // This test crashes in Jenkins about 5% of the time.
-    if (!Cypress.env('CIRCLECI')) this.skip();
+    // /if (!Cypress.env('CIRCLECI')) this.skip();
```
Comment (mpelzsherman) [inline src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: Please discus this change with the testing tools team, esp. @ddzz

Comment (brangi) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: I am not planning to just enable it, however, I want find out if this is correct `crashing on Jenkins about one out of every 15-20 runs.` first

Comment (ddzz) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: @brangi I was able to recreate this failure locally and on Jenkins by running the test in a loop. There's an example in the [Cypress flaky test management document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-flaky-test-management.md#how-to-detect-flaky-tests).

Comment (brangi) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: @ddzz , oh thanks for this, let me try it out

Comment (brangi) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: @ddzz what do you mean by https://github.com/department-of-veterans-affairs/va.gov-team/issues/20178#issuecomment-781679897 `reduce map animations ` ? you mean for example when the map zooms in slowly (navigating to the location)

Comment (brangi) [reply src/applications/facility-locator/tests/e2e/search.cypress.spec.js:34]: AFAIK we are already Mocking Mapbox API calls https://github.com/department-of-veterans-affairs/vets-website/blob/947e2bb66414c0376ae9213bf6350e791e751ba8/src/applications/facility-locator/tests/e2e/search.cypress.spec.js#L49 - are we not?

**Pull request closed without merging**